### DISCUSSION TITLE
How do I inspect or understand the structure of search results without deploying and logging?

### DISCUSSION THREAD

**Discussion by andhines** [canvas-medical/canvas-plugins#115] [Q&A]
I'm following the [Customize Search Results](https://docs.canvasmedical.com/guides/customize-search-results/) guide and tweaking for prescribe medication search not med statement. I need to understand the structure of items in the `results` iterable, and I want to do it without deploying and logging. (I can't seem to see print statements in the logs, but that's another matter.) 

How can I gain the understanding I'm after, so I can actually work with the objects? Right now I just copy-paste example code and don't know what's going on or what else I could potentially work with, e.g. what codings are available to me, etc.

**Comment by marythought** [comment]
Hi! This is a great question/suggestion -- you probably have all the objects memorized by now 😉 but for anyone getting started this is a preventable barrier. We've added a ton of information to our docs site since your initial question, for examples the [events page](https://docs.canvasmedical.com/sdk/events/) gives a basic data contract for actions that happen within Canvas, but there is still room for improvement. I've added an [issue](https://github.com/canvas-medical/documentation/issues/587) here for tracking and look forward to your feedback as we iterate. 🙏
